src/tree_data.h

```diff
@@ -770,6 +771,16 @@ struct lyd_node *lyd_child_no_keys(const struct lyd_node *node);
  */
 const struct lys_module *lyd_owner_module(const struct lyd_node *node);
 
+/**
+ * @brief Check whether this data node existence depends on any when conditions. Be careful, this function is
+ * not recursive and checks only conditions that apply to this node directly. Meaning if there are any conditions
+ * associated with any parent of @p node, they are not returned.
```

Comment:
maybe noticing why just checking `node->schema->whne` is not enough would be a good motivation to use this function

Reply:
Fixed.